pkg/cloud/aws/actuators/machine/actuator.go

```diff
@@ -56,6 +63,7 @@ func NewActuator(params ActuatorParams) *Actuator {
 		res.servicesGetter = new(defaultServicesGetter)
 	}
 
+	res.Deployer = deployer.New(res.servicesGetter)
```

Comment:
I'm not really digging our runtime dependency on the aws region, but there's not much else to do here. It feels like an extra layer to mock through during tests.

Reply:
Yeah, this is a requirement though because the clusters might be in a different region than the one we're currently operating in